setup.cfg

```diff
@@ -19,8 +19,8 @@ classifiers =
     Programming Language :: JavaScript
 
 [options]
-packages = find:
-include_package_data = True
+packages = jupyterhub_moss
+include_package_data = False
```

Comment:
Automatic data include was the source of the warnings, so now the list of resources is explicit (below)